aptos-move/framework/aptos-framework/sources/reconfiguration.move

```diff
@@ -15,7 +15,6 @@ module aptos_framework::reconfiguration {
     friend aptos_framework::aptos_governance;
     friend aptos_framework::block;
     friend aptos_framework::consensus_config;
-    friend aptos_framework::features;
```

Comment:
This was not necessary and was causing some weird cyclical dependencies when building `cached-packages` (for reasons I couldn't understand)